.github/workflows/ci.yml

```diff
@@ -12,6 +12,30 @@ jobs:
         run: |
           pip install vim-vint
           pip install pre-commit
-      - name: Run
+      - name: Run Vint
         run: |
           pre-commit run --all-files || { git diff --color=always && exit 1; }
+  unit-tests:
+    name: Unit Tests
+    runs-on: ubuntu-20.04
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Vim
+        uses: rhysd/action-setup-vim@v1.2.6
+        with:
+          neovim: true
+      - name: Ensure plugins directory exists
+        run: |
+          mkdir -p $HOME/.config/nvim/plugins
+      - name: Create a fake code-minimap executable
+        # Necessary for minimap.vim to start up
+        run: |
+          mkdir -p "$HOME/bin"
+          ln -s /bin/echo "$HOME/bin/code-minimap"
+          echo "$HOME/bin" >> $GITHUB_PATH
```

Comment:
What does this line do? Equivalent to append `$HOME/bin` to the `PATH` environment variable?

Reply:
Yes, it updates the `PATH`. I think `GITHUB_PATH` is a bit different in that it is available for every subsequent step without needing to re-source anything. When I looked up how to update the `PATH` in the docs, this was the recommended way so I just went with it.

Reply:
Thank you for the explanation @ZNielsen